zbus/src/blocking/proxy_builder.rs

```diff
@@ -6,7 +6,7 @@ use zvariant::ObjectPath;
 
 use crate::{blocking::Connection, utils::block_on, CacheProperties, Error, Result};
 
-pub use crate::ProxyDefault;
+pub use crate::proxy::ProxyDefault;
```

Comment:
OT but I was discussing this type and proxy building API in general with someone and I think the current API is a bit weird. I think user should always be able to use the builder pattern, whether they are using the macro-generated proxies or `Proxy`: `Proxy::builder()`. The macros can add the constants into generated proxy impl directly and we can ditch this type (no need to even deprecate since I seriously doubt anyone uses the trait itself) and unify `Builder::{new, new_bare}` as `Builder::new`.